packages/cli/test/unit/commands/promote/index.test.ts

```diff
@@ -135,9 +137,32 @@ describe('promote', () => {
     });
 
     it('should promote a preview deployment when user says yes', async () => {
+      const inspectorUrl = `https://vercel.com/user/${projectName}/abcdefghijlmo`;
+
       const { cwd, previousDeployment } = initPromoteTest({
         deploymentTarget: 'preview',
       });
+      let createWasCalled = false;
+
+      client.scenario.post(
```

Comment:
Placed the scenario in-line because I wanted to assert the exact body submitted as well.